components/password-reset-request-modal-component.html

```diff
@@ -0,0 +1,60 @@
+<br-modal br-title="{{$ctrl.title}}">
+  <br-modal-body>
+    <div ng-if="$ctrl.display.requestForm">
+      <h4 ng-if="!$ctrl.readOnly" class="col-md-offset-1">
+        Enter your email address and we will send you a link to set your password.
+      </h4>
+      <h4 ng-if="$ctrl.readOnly" class="col-md-offset-1">
+        We will send you an email with a link to set your password.
+      </h4>
+      <form name="$ctrl.requestForm" class="form-horizontal">
+        <fieldset>
+          <br-input br-model="$ctrl.sysIdentifier"
+            br-options="{
+              name: 'sysIdentifier',
+              label: 'Email',
+              showValidation: true,
+              type: 'email',
+              readonly: '{{$ctrl.readOnly}}',
+              columns: {
+                label: 'col-md-2',
+                input: 'col-md-9',
+                help: 'col-md-offset-2 col-md-9',
+                validation: 'col-md-offset-2 col-md-9'
+              }
+            }" required autofocus>
```

Comment:
Change `required` to `br-input-required` and `autofocus` to `br-input-autofocus`. This will transplant those to the `input` element once generated by the br-input component.